R/motif_enrichment_HOMER.R

```diff
@@ -95,6 +96,16 @@ dumpJaspar <- function(filename,
     # load PFMs and convert to HOMER format (base probabilties)
     requireNamespace(pkg)
     mdb <- utils::getFromNamespace(pkg, ns = pkg)
+    if (!methods::existsMethod(f = getMatrixSet, signature = list(x = class(mdb)))) {
+        # workaround for JASPAR2024 (no specific TFBSTools::getMatrixSet method yet)
+        if (inherits(mdb, "function") && exists(x = "db", where = paste0("package:", pkg))) {
```

Comment:
This will fail if `JASPAR2024` is not loaded - can we assume that it will be? 

```
> exists(x = "db", where = paste0("package:", pkg))
Error in as.environment(where) : 
  no item called "package:JASPAR2024" on the search list
```

Reply:
You are right - I replaced the calls by `exists("db", envir = asNamespace(pkg))` and `getFromNamespace(x = "db", ns = pkg)`